### PR TITLE
Vendor gardener/gardener@v1.12.5

### PR DESCRIPTION
**Pull request by ialidzhikov**
**Release note**:

```improvement developer
`github.com/gardener/gardener` dependency is now updated to `v1.12.5`.
```

### CODE REVIEW

**Review by timebertt — APPROVED** [review]
/lgtm

**Comment by timebertt** [comment]
```
> Generate / Vendor Check
>> make generate
>> make revendor
make revendor needs to be run:
 M vendor/modules.txt
```

File: go.mod [hunk 1]
```diff
@@ -1,24 +1,26 @@
 module github.com/gardener/gardener-extension-os-ubuntu
 
-go 1.14
+go 1.15
 
 require (
-	github.com/gardener/gardener v1.10.0
+	github.com/gardener/gardener v1.12.5
```
Comment (timebertt) [inline go.mod:6]: Hmm, where are the github templates?
Can you add a script like this https://github.com/gardener/gardener-extension-provider-aws/blob/master/hack/update-github-templates.sh?

**Review by rfranzke — APPROVED** [review]
/lgtm

**Pull request merged**